docker-compose.yml

```diff
@@ -52,11 +49,8 @@ services:
     image: redis:3.2
 
   kdc-elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:5.4.1
+    image: docker.elastic.co/elasticsearch/elasticsearch:5.6.10
```

Comment:
We aren't using `kuzzleio/elasticsearch` anymore ?

Reply:
I believe our es image is deprecated

Reply:
I'm not quite sure, @benoitvidis just updated the image to use the last elasticsearch version 10 days ago https://github.com/kuzzleio/kuzzle-containers/tree/master/elasticsearch

Reply:
ah, you seem to be right
I'll fix that, then